airbyte-integrations/connectors/source-sentry/source_sentry/manifest.yaml

```diff
@@ -1,105 +1,175 @@
+    cursor_datetime_formats:
+      - "%Y-%m-%dT%H:%M:%S.%f%z"
+      - "%Y-%m-%dT%H:%M:%SZ"
+    datetime_format: "%Y-%m-%dT%H:%M:%S.%f%z"
+    start_datetime:
+      type: MinMaxDatetime
+      datetime: "1900-01-01T00:00:00.0Z"
+      datetime_format: "%Y-%m-%dT%H:%M:%S.%fZ"
+    is_data_feed: true
+  base_stream_full_refresh:
     schema_loader:
       $ref: "#/definitions/schema_loader"
     retriever:
       $ref: "#/definitions/retriever"
-      record_selector:
-        $ref: "#/definitions/selector"
+  base_stream_incremental:
+    schema_loader:
+      $ref: "#/definitions/schema_loader"
+    retriever:
+      $ref: "#/definitions/retriever_with_filter"
+    incremental_sync:
+      $ref: "#/definitions/incremental_sync"
+
+  # Stream Events https://docs.sentry.io/api/events/list-a-projects-error-events/
+  events:
+    type: DeclarativeStream
+    $parameters:
+      name: "events"
+      primary_key: "id"
+      path: "projects/{{ config['organization'] }}/{{ config['project'] }}/events/"
+      cursor_field: "dateCreated"
+    retriever:
+      type: SimpleRetriever
       requester:
         $ref: "#/definitions/requester"
-        path: "projects/{{config.organization}}/{{config.project}}/events/"
         request_parameters:
           full: "true"
+      record_selector:
+        $ref: "#/definitions/record_selector"
+        record_filter:
+          condition: "{{ record[parameters['cursor_field']] > stream_state.get(parameters['cursor_field'], '') }}"
       paginator:
         $ref: "#/definitions/paginator"
-  - type: DeclarativeStream
+    incremental_sync:
+      $ref: "#/definitions/incremental_sync"
+
+  # Stream Issues https://docs.sentry.io/api/events/list-a-projects-issues/
+  issues:
+    type: DeclarativeStream
     $parameters:
       name: "issues"
-    primary_key: "id"
-    schema_loader:
-      $ref: "#/definitions/schema_loader"
+      primary_key: "id"
+      path: "projects/{{ config['organization'] }}/{{ config['project'] }}/issues/"
+      cursor_field: "lastSeen"
     retriever:
-      $ref: "#/definitions/retriever"
-      record_selector:
-        $ref: "#/definitions/selector"
+      type: SimpleRetriever
       requester:
         $ref: "#/definitions/requester"
-        path: "projects/{{config.organization}}/{{config.project}}/issues/"
         request_parameters:
           statsPeriod: ""
-          query: ""
+          query: "lastSeen:>{{ stream_state.get(parameters['cursor_field']) or '1900-01-01T00:00:00.0Z' if stream_state else '1900-01-01T00:00:00.0Z' }}"
+      record_selector:
+        $ref: "#/definitions/record_selector"
       paginator:
         $ref: "#/definitions/paginator"
-  - type: DeclarativeStream
+    incremental_sync:
+      $ref: "#/definitions/incremental_sync"
+
+  # Stream Projects https://docs.sentry.io/api/projects/list-your-projects/
+  projects:
+    type: DeclarativeStream
+    $ref: "#/definitions/base_stream_incremental"
```

Comment:
`projects` was not fully incremental but only semi-incremental. This is not supported in the current low-code CDK hence I don't think this would work as the API does not allow for filtering using query params.